### DISCUSSION TITLE
WEEKLY REPORTS?

### DISCUSSION THREAD

**Discussion by ITgeevoSOC** [Security-Onion-Solutions/securityonion#5762] [Unsupported Versions] [answered]
Is there an option to download weekly reports from critical alerts on Security Onion?

**Comment by dougburks** [comment, marked as answer]
Depending on what you're trying to do, Kibana has some reporting functionality or you could put together a script to query Elasticsearch and generate a custom report.

**Reply by ITgeevoSOC** [reply]
Hi Doug,

Thank you for your response.

How can i generate reports on Kibana? Do i need to configure something beforehand in order to proceed?

**Reply by dougburks** [reply]
Have you tried googling for "Kibana reports"?

**Reply by ITgeevoSOC** [reply]
Thank you. Found a solution.

**Reply by domlaz11** [reply]
would you share what you found?

**Reply by ITgeevoSOC** [reply]
We found a third party vendor as seen from the link below:

https://www.skedler.com/

**Reply by achak17** [reply]
Skedler is the most convenient and affordable report automation tool for Kibana and Grafana. 

You can generate Security Onion reports using Skedler's integration with Kibana. 

Refer to this blog to check out the easy steps to automate Security Onion reports: https://www.skedler.com/blog/automate-security-onion-reports-using-skedler/
